### DISCUSSION TITLE
Need help with a writing project.

### DISCUSSION THREAD

**Discussion by hernessteve** [NAIST-SE/AttractingContributors#324] [General]
English is not my native language and I am really worried about it. I'm not pretty sure if I write correctly. Need some app or site to check my grammar and spelling. Any recommendations?

**Comment by ellielo** [comment]
Just use Grammar for checking the mistakes.

**Comment by Ozankey** [comment]
There are plenty of online writing services. When I was studying in school, I also used them when I had little time or couldn't write assignments myself. I read good reviews about https://letsgradeit.com/review/allassignmenthelp/ and worked with them before. You can address them to help you to write or just check your work.

**Comment by ellielo** [comment]
Thank you all for your answers

**Comment by XelaAl** [comment]
No worries, your English looks good! For grammar and spelling, try using tools like Grammarly or Hemingway. They’re great for catching errors and improving writing. Also, [resume now](https://resume-now.pissedconsumer.com/review.html) is a helpful site for polishing your resume and other documents. It might boost your confidence in writing. Hope this helps!
